lms/templates/financial-assistance/financial_assessment_form.underscore

```diff
@@ -7,10 +7,8 @@
 </div>
 
 <form class="financial-assistance-form" method="POST">
-	<div class="status submission-error hidden" aria-live="polite">
-	    <h4 class="message-title"><%- gettext('Unable to submit application') %></h4>
-	    <ul class="message-copy"></ul>
-	</div>
+  <div class="js-form-feedback" aria-live="assertive" tabindex="-1">
```

Comment:
Yes, we decided to go with `assertive`, and also to set focus to the feedback container instead of the first error field. That's why I removed the `focusFirstError` function.